### PR TITLE
Samsung Internet, Updating U-Z files, in api/

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
A checklist to help your pull request get merged faster:
- [ ] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

File: api/URL.json [hunk 1]
```diff
@@ -78,9 +78,15 @@
               "prefix": "webkit"
             }
           ],
-          "samsunginternet_android": {
-            "version_added": true
-          },
+          "samsunginternet_android": [
+            {
+              "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/URL.json]:
```suggestion
              "version_added": "2.0"
```

File: api/WEBGL_debug_renderer_info.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
             "version_added": null
           },
           "samsunginternet_android": {
-            "version_added": null
+            "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/WEBGL_debug_renderer_info.json]:
```suggestion
            "version_added": "2.0"
```

File: api/WebGLActiveInfo.json [hunk 3]
```diff
@@ -35,7 +35,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLActiveInfo.json]:
```suggestion
            "version_added": "1.5"
```

File: api/WebGLActiveInfo.json [hunk 4]
```diff
@@ -82,7 +82,7 @@
               "version_added": "8"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLActiveInfo.json]:
```suggestion
              "version_added": "1.5"
```

File: api/WebGLActiveInfo.json [hunk 5]
```diff
@@ -130,7 +130,7 @@
...
```
Comment (vinyldarkscratch) [inline api/WebGLActiveInfo.json]:
```suggestion
              "version_added": "1.5"
```

File: api/WebGLActiveInfo.json [hunk 6]
```diff
@@ -178,7 +178,7 @@
...
```
Comment (vinyldarkscratch) [inline api/WebGLActiveInfo.json]:
```suggestion
              "version_added": "1.5"
```

File: api/WebGLBuffer.json [hunk 7]
```diff
@@ -35,7 +35,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLBuffer.json]:
```suggestion
            "version_added": "1.5"
```

File: api/WebGLContextEvent.json [hunk 8]
```diff
@@ -35,7 +35,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLContextEvent.json]:
```suggestion
            "version_added": "1.5"
```

File: api/WebGLContextEvent.json [hunk 9]
```diff
@@ -82,7 +82,7 @@
               "version_added": "8"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLContextEvent.json]:
```suggestion
              "version_added": "1.5"
```

File: api/WebGLFramebuffer.json [hunk 10]
```diff
@@ -35,7 +35,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLFramebuffer.json]:
```suggestion
            "version_added": "1.5"
```

File: api/WebGLProgram.json [hunk 11]
```diff
@@ -35,7 +35,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLProgram.json]:
```suggestion
            "version_added": "1.5"
```

File: api/WebGLRenderbuffer.json [hunk 12]
```diff
@@ -35,7 +35,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLRenderbuffer.json]:
```suggestion
            "version_added": "1.5"
```

File: api/WebGLRenderingContext.json [hunk 13]
```diff
@@ -37,7 +37,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLRenderingContext.json]: (All of the changes in this file should be "1.5" instead, but there's just so many.)

File: api/WebGLShader.json [hunk 14]
```diff
@@ -35,7 +35,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLShader.json]:
```suggestion
            "version_added": "1.5"
```

File: api/WebGLShaderPrecisionFormat.json [hunk 15]
```diff
@@ -35,7 +35,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLShaderPrecisionFormat.json]:
```suggestion
            "version_added": "1.5"
```

File: api/WebGLShaderPrecisionFormat.json [hunk 16]
```diff
@@ -82,7 +82,7 @@
               "version_added": "8"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLShaderPrecisionFormat.json]:
```suggestion
              "version_added": "1.5"
```

File: api/WebGLShaderPrecisionFormat.json [hunk 17]
```diff
@@ -130,7 +130,7 @@
...
```
Comment (vinyldarkscratch) [inline api/WebGLShaderPrecisionFormat.json]:
```suggestion
              "version_added": "1.5"
```

File: api/WebGLShaderPrecisionFormat.json [hunk 18]
```diff
@@ -178,7 +178,7 @@
...
```
Comment (vinyldarkscratch) [inline api/WebGLShaderPrecisionFormat.json]:
```suggestion
              "version_added": "1.5"
```

File: api/WebGLTexture.json [hunk 19]
```diff
@@ -35,7 +35,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLTexture.json]:
```suggestion
            "version_added": "1.5"
```

File: api/WebGLUniformLocation.json [hunk 20]
```diff
@@ -35,7 +35,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/WebGLUniformLocation.json]:
```suggestion
            "version_added": "1.5"
```

File: api/Window.json [hunk 21]
```diff
@@ -1486,7 +1536,7 @@
               "version_added": "6.1"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
              "version_added": "3.0"
```

File: api/Window.json [hunk 22]
```diff
@@ -2457,7 +2510,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
              "version_added": "3.0"
```

File: api/Window.json [hunk 23]
```diff
@@ -2520,7 +2573,7 @@
...
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
              "version_added": "3.0"
```

File: api/Window.json [hunk 24]
```diff
@@ -3560,7 +3619,7 @@
                 "version_added": "7"
               },
               "samsunginternet_android": {
-                "version_added": null
+                "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
                "version_added": "2.0"
```

File: api/Window.json [hunk 25]
```diff
@@ -4753,9 +4818,16 @@
             "safari_ios": {
               "version_added": false
             },
-            "samsunginternet_android": {
-              "version_added": true
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
                "version_added": "3.0"
```

File: api/Window.json [hunk 26]
```diff
@@ -4753,9 +4818,16 @@
...
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Window.json [hunk 27]
```diff
@@ -4753,9 +4818,16 @@
...
+                "version_removed": "4.0"
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
                "version_removed": "3.0"
```

File: api/Window.json [hunk 28]
```diff
@@ -4829,9 +4901,16 @@
             "safari_ios": {
               "version_added": false
             },
-            "samsunginternet_android": {
-              "version_added": true
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
                "version_added": "3.0"
```

File: api/Window.json [hunk 29]
```diff
@@ -4829,9 +4901,16 @@
...
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Window.json [hunk 30]
```diff
@@ -4829,9 +4901,16 @@
...
+                "version_removed": "4.0"
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
                "version_removed": "3.0"
```

File: api/Window.json [hunk 31]
```diff
@@ -6977,9 +7069,15 @@
                 "prefix": "webkit"
               }
             ],
-            "samsunginternet_android": {
-              "version_added": true
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
                "version_added": "1.5"
```

File: api/Window.json [hunk 32]
```diff
@@ -7025,7 +7123,7 @@
                 "version_added": "6.1"
               },
               "samsunginternet_android": {
-                "version_added": null
+                "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/Window.json]:
```suggestion
                "version_added": "1.5"
```

File: api/WorkerNavigator.json [hunk 33]
```diff
@@ -82,7 +82,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/WorkerNavigator.json]:
```suggestion
              "version_added": "3.0"
```

File: api/XMLHttpRequest.json [hunk 34]
```diff
@@ -1268,7 +1286,7 @@
               "version_added": true
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/XMLHttpRequest.json]:
```suggestion
              "version_added": "3.0"
```

File: api/WebGLRenderingContext.json (hunk 13 above)
Comment (AdaRoseCannon) [reply api/WebGLRenderingContext.json]: Done

File: api/URL.json [hunk 35]
```diff
@@ -803,7 +804,7 @@
               "version_added": "6"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "1.0"
```
Comment (vinyldarkscratch) [inline api/URL.json]:
```suggestion
              "version_added": "1.5"
```

File: api/URL.json [hunk 36]
```diff
@@ -205,7 +205,7 @@
...
```
Comment (vinyldarkscratch) [inline api/URL.json]:
```suggestion
              "version_added": "1.5"
```

**Pull request merged**